packages/share_plus/share_plus_platform_interface/test/share_plus_platform_interface_test.dart

```diff
@@ -98,7 +98,7 @@ void main() {
         text: 'some text to share',
         sharePositionOrigin: const Rect.fromLTWH(1.0, 2.0, 3.0, 4.0),
       );
-      verify(mockChannel.invokeMethod<void>(
+     await verifyNever(mockChannel.invokeMethod<void>(
```

Comment:
Maybe I'm just blind, but why do you have all these changes in the tests? If we always say `verifyNever` what does that actually test?  If your changes override plugin chanel behaviour, please ensure that the test still cover the `mockChannel` on Windows etc..

Please correct me if I am overlooking something :)